### ISSUE TITLE
take_while_view::sentinel declaration is missing the Pred parameter

### ISSUE THREAD

**Issue by zygoloid** [cplusplus/draft#3666]
We have

```
namespace std::ranges {
  template<view V, class Pred>
  requires input_range<V> && is_object_v<Pred> &&
  indirect_unary_predicate<const Pred, iterator_t<V>>
class take_while_view
```

but

```
namespace std::ranges {
    template<class V>
    template<bool Const>
    class take_while_view<V>::sentinel {
```

(missing the second template argument and the requires-clause).

**Comment by CaseyCarter** [comment]
Ack! Thanks for the reminder: this problem is endemic throughout [ranges]. Feel free to assign to me.

**Comment by johelegp** [comment]
This was fixed: https://github.com/cplusplus/draft/pull/3604/files.

**Comment by CaseyCarter** [comment]
> This was fixed: https://github.com/cplusplus/draft/pull/3604/files.

That adds the template parameter `Pred`, but not the missing *type-constraint* and *requires-clause*.

**Comment by zygoloid** [comment]
Fixed in 85121a254c5f013af41a305ad008586a595b208e.
